### ISSUE TITLE
How to set 'train / traditional', 'train / CNN' and 'train / mix'?

### ISSUE THREAD

**Issue by zhfewag** [saikatdutta/Stacked_DMSHN_bokeh#2]
Hello, I'd like to ask. How do you set up'train/traditional','train/cnn' and'train/mix'here? Under that folder? Is it the dataset set of AIM2020?

**Comment by saikatdutta** [comment]
I didn't understand the query. Where did you get 'train/traditional' etc.? 
We use EBB dataset which is the same dataset used in AIM 2020.

**Comment by zhfewag** [comment]
In the perceptual similarity folder of your warehouse, click the file train. Py. When training here, you need to set the datasets, that is, line 13 of the code,：parser. Add_ Argument ('-- datasets', type = STR, nargs =' + ', default = ['train / traditional','train / CNN ','train / mix'], help='datasets to train on: [train / traditional], [train / CNN], [train / mix], [Val / traditional], [Val / CNN], [Val / color], [Val / blur], [Val / frameinterval], [Val / supperres] '). Here, the default value of train / traditional refers to?

**Comment by zhfewag** [comment]
In the perceptual similarity folder of your warehouse, click the file train. Py. When training here, you need to set the datasets, that is, line 13 of the code,：parser. Add_ Argument ('-- datasets', type = STR, nargs =' + ', default = ['train / traditional','train / CNN ','train / mix'], help='datasets to train on: [train / traditional], [train / CNN], [train / mix], [Val / traditional], [Val / CNN], [Val / color], [Val / blur], [Val / frameinterval], [Val / supperres] '). Here, the default value of train / traditional refers to?
I have carefully read your article: depth aware blending of smooth images for bokeh effect generation. Here, you propose to divide aim2019 training set into 4400 and 294 parts. Is 294 here the same as your current verification set?
Thank you very much for your answer.

**Comment by saikatdutta** [comment]
Hi, Perceptual similarity folder is taken from https://github.com/richzhang/PerceptualSimilarity
The only purpose of this folder is to calculate the LPIPS score (refer to eval.py in main directory). We have not provided training code for DMSHN/Stacked DMSHN in this repo.

Yes, the validation set used in this paper is same as used in "Depth aware blending of smooth images for bokeh effect generation". Thank you for showing interest in our works!

**Comment by zhfewag** [comment]
Thank you very much for your patient reply. I am still an undergraduate, and I am very interested in your work. I wonder if you can share your training code? I just want to learn. My email address is: zhifengwang686@gmail.com. If it's not convenient, it doesn't matter. Thank you.

**Comment by zhfewag** [comment]
Thank you very much for your patient reply. I am still an undergraduate, and I am very interested in your work. I wonder if you can share your training code? I just want to learn. My email address is: zhifengwang686@gmail.com. If it's not convenient, it doesn't matter. Thank you.
